timify-api/server.js

```diff
@@ -13,7 +13,7 @@ import SequelizeStoreInit from 'connect-session-sequelize'
 
 const app = express()
 
-const sessionTime = 365 * 24 * 60 * 60 * 1000 // 1 year in milliseconds
+const sessionTime = 365 * 24 * 60 * 60 * 1000
```

Comment:
I think if you're going to remove this comment you should use the variable name to make it clear this is 1 year. the goal is readability